### DISCUSSION TITLE
Border of images and other elements are getting wrong positions

### DISCUSSION THREAD

**Discussion by saharhagekr** [GrapesJS/grapesjs#5285] [Q&A]
On smaller laptop screens (macbooks for example), the border of images and some elements are in the wrong position (when i click to move or resize an image)
I am not having this issue when the editor gets rendered on bigger screens.

I am not sure if there is a config step that i'm missing.

Can anyone please assist? Is this a common issue?

For example:
<img width="217" alt="Screenshot 2023-08-08 at 3 20 19 PM" src="https://github.com/GrapesJS/grapesjs/assets/121887023/9b83c4a2-c944-4faf-94e0-3d6bca9c80d1">
<img width="245" alt="Screenshot 2023-08-08 at 3 20 25 PM" src="https://github.com/GrapesJS/grapesjs/assets/121887023/8526ca87-e312-4fdb-ae1e-92799398805d">

**Comment by artf** [comment]
I'd expect no related issues with smaller screens, is it reproducible on the official demo?

**Reply by saharhagekr** [reply]
No i cannot reproduce it on the official demo. Although i have the same configuration set. I am not sure if i'm missing any step ...

**Reply by artf** [reply]
You have probably a layout shift post-render of the editor which causes that issue. In case you're moving the layout of the editor you should call [`editor.refresh()`](https://grapesjs.com/docs/api/editor.html#refresh) to update the position of the toolbars

**Reply by saharhagekr** [reply]
I am not moving the layout at all. I have already tried to call editor.refresh() but it doesn't seem to work.

**Reply by saharhagekr** [reply]
Hello, any update on this? I haven't figured it out yet. I am not moving the layout + editor.refresh() and any other solutions that i've tried are not working

**Reply by artf** [reply]
@saharhagekr can you provide a reproducible demo (use this [starter](https://jsfiddle.net/szLp8h4n))?
